tests/test_style/test_rcmod.py

```diff
@@ -38,9 +38,25 @@ class RCParamTester(VisualTestCase):
     """
 
     excluded_params = {
-        "backend",  # This cannot be changed by manipulating rc
+        "backend",  # This cannot be changed by manipulating rc,
         "svg.embed_char_paths",  # This param causes test issues and is deprecated
         "font.family",  # breaks the visualtest case
+
+        # The following are not able to be set by a style in matplotlib 2.0
+        # See: https://bit.ly/39UWTCY
+        "interactive",
+        "backend.qt4",
+        "webagg.port",
+        "webagg.port_retries",
+        "webagg.open_in_browser",
+        "backend_fallback",
+        "toolbar",
+        "timezone",
+        "datapath",
+        "figure.max_open_warning",
+        "savefig.directory",
+        "tk.window_focus",
+        "docstring.hardcopy",
```

Comment:
@rebeccabilbro did a lot of work regenerating baseline images for binning, class balance, feature correlation, dispersion plot, freqdist, postag, and tsne - but somehow adding these lines to the exclude function managed to revert them to their original state (well fingers crossed, it worked on my MacBook, but not sure about CI). The changes seemed to be related to grid lines and bar colors. 

current working hypothesis - we were setting one of these values unintentionally and as a result, matplotlib was ignoring all of our styles, by excluding them, the styles work as expected? 

But you know … that's just a hypothesis, I really have no clue what is actually happening.

Reply:
@rebeccabilbro I did build the docs and everything was fine - and the figures generated by the plot directive were as I expected, RE grid lines and colors.

Reply:
Still flummoxed by this … 🤷